invenio_app_rdm/administration/moderation/requests.py

```diff
@@ -0,0 +1,161 @@
+
+    display_search = True
+    display_delete = False
+    display_create = False
+    display_edit = False
+    display_read = False
+
+    search_config_name = "REQUESTS_MODERATION_REQUEST_SEARCH"
+    search_facets_config_name = "REQUESTS_MODERATION_REQUEST_FACETS"
+    search_sort_config_name = "REQUESTS_MODERATION_REQUEST_SORT_OPTIONS"
+
+    item_field_list = {
+        "type": {"text": _("Type"), "order": 1, "width": 3},
+        "created": {"text": _("Created"), "order": 2, "width": 3},
+        "last_reply": {"text": _("Last reply"), "order": 3, "width": 3},
+    }
+
+    @property
+    def request_type(self):
+        """Request type property."""
+        request_type = self.resource.service.request_type_registry.lookup(
+            "record-deletion"
+        )
+        return request_type
+
+    @classmethod
+    def get_service_schema(cls):
+        """Get marshmallow schema of the assigned service."""
+        request_type = cls.resource.service.request_type_registry.lookup(
+            "record-deletion"
+        )
+        # handle dynamic schema class declaration for requests
+        schema_cls = request_type.marshmallow_schema()
+        schema = schema_cls()
+        return schema
+
+    @staticmethod
+    def disabled():
+        """Disable the view on demand."""
+        return False
+
+    def init_search_config(self):
+        """Build search view config."""
+        return partial(
+            search_app_config,
+            config_name=self.get_search_app_name(),
+            available_facets=current_app.config.get(self.search_facets_config_name),
+            sort_options=current_app.config[self.search_sort_config_name],
+            endpoint=self.get_api_endpoint(),
+            headers=self.get_search_request_headers(),
+            initial_filters=[["is_open", "true"]],
+            hidden_params=[
+                ["expand", "1"],
+                ["type", RecordDeletion.type_id],
+            ],
+            pagination_options=(20, 50),
+            default_size=20,
+        )
+
+
+class ModerationRequestDetailView(AdminResourceDetailView):
+    """Configuration for moderation request detail view."""
+
+    url = "/requests/<pid_value>"
+    extension_name = "invenio-requests"
+    api_endpoint = "/requests"
+    name = "request_details"
+    resource_config = "requests_resource"
+    title = _("Request details")
+    display_delete = False
+    display_edit = False
+
+    pid_path = "id"
+    request_headers = {"Accept": "application/json"}
+    template = "invenio_app_rdm/administration/requests_details.html"
+
+    @classmethod
+    def get_service_schema(cls):
+        """Get marshmallow schema of the assigned service."""
+        request_type = cls.resource.service.request_type_registry.lookup(
+            "record-deletion"
+        )
+        # handle dynamic schema class declaration for requests
+        schema_cls = request_type.marshmallow_schema()
+        schema = schema_cls()
+        return schema
```

Comment:
This probably can be removed since we're not auto-generating the details view page.